sdk/packages/selector-base/src/types.ts

```diff
@@ -19,6 +19,15 @@ export interface MetadataWallet {
   walletType: WalletType
 }
 
+export interface WalletMetadata extends Partial<WalletMetadataOriginal> {
+  slug: string
+  name: string
+  walletType: WalletType
+  mobile: Deeplink | null
+  desktop: Deeplink | null
+  image: Images
+}
+
```

Comment:
WHAT ? remove this

Reply:
But wouldnt it then force me to pass all of the fields such ass homepage, version, app etc. in for example polkadot selector detection file?
`walletsData[wallet.name.toLocaleLowerCase()] = {
      slug: wallet.name,
      name: wallet.name,
      image: {
        default: wallet.icon as string,
        sm: wallet.icon as string,
        md: wallet.icon as string,
        lg: wallet.icon as string
      },
      desktop: null,
      mobile: null,
      recent: recentWalletName === wallet.name,
      detected: true,
      injectedWallet: wallet,
      walletType: 'hybrid'
    }`

Reply:
But wouldnt it then force me to pass all of the fields such as homepage, version, app etc. in for example polkadot selector detection file?
`    walletsData[wallet.name.toLocaleLowerCase()] = {
      slug: wallet.name,
      name: wallet.name,
      image: {
        default: wallet.icon as string,
        sm: wallet.icon as string,
        md: wallet.icon as string,
        lg: wallet.icon as string
      },
      desktop: null,
      mobile: null,
      recent: recentWalletName === wallet.name,
      detected: true,
      injectedWallet: wallet,
      walletType: 'hybrid'
    }`